modules/OutboundEmailAccounts/OutboundEmailAccounts.php

```diff
@@ -51,7 +56,12 @@ public function save($check_notify = false) {
 		if(!$this->mail_smtppass && $this->id) {
 			$bean = new OutboundEmailAccounts();
 			$bean->retrieve($this->id);
-			$this->mail_smtppass = $bean->mail_smtppass;
+			if(!$bean->mail_smtppass) {
+				$GLOBALS['log']->warn("No mail SMTP password");
```

Comment:
Can we change the message to something like "Unable to send email via SMTP using an empty password. Please ensure that the email settings are configured correctly"?